### PR TITLE
updates to release tests

### PR DESCRIPTION
**Pull request by jlewitt1**

### CODE REVIEW

**Comment by jlewitt1** [comment]
* **#1320** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1320?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jlewitt1 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .github/workflows/nightly_release_testing.yaml [hunk 1]
```diff
@@ -247,6 +247,7 @@ jobs:
       - ondemand-aws-tests
       - ondemand-gcp-tests
       - kubernetes-tests
+      - ondemand-aws-multinode-tests
```
Comment (carolineechen) [inline .github/workflows/nightly_release_testing.yaml:250]: is this job defined somewhere? I can't seem to find it. and do we want to aws w/ https auth to nightly release testing as well?

Comment (jlewitt1) [reply .github/workflows/nightly_release_testing.yaml:250]: yea its defined lower down in the file, basically we're using it to run the tests inside `TestMultiNodeCluster`

the ondemand aws tests should be running with right now actually, using the `ondemand_aws_https_cluster_with_auth` fixture

Comment (carolineechen) [reply .github/workflows/nightly_release_testing.yaml:250]: oops my local repo has not been updated in 2 weeks, looks good!

**Review by carolineechen — APPROVED** [review]

**Pull request merged**